server/metrics/metrics.go

```diff
@@ -906,6 +917,40 @@ var (
 		Help:      "Per-file upload duration during remote execution, in **microseconds**.",
 	})
 
+	FirecrackerStageDurationUsec = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: bbNamespace,
+		Subsystem: "firecracker",
+		Name:      "stage_duration_usec",
+		Buckets:   durationUsecBuckets(1*time.Millisecond, 1*day, 10),
+		Help:      "The total duration of each firecracker stage, in microseconds.",
+	}, []string{
+		Stage,
+		StatusHumanReadableLabel,
+		RecycledRunnerStatus,
+		GroupID,
+		SnapshotSharingStatus,
+	})
+
+	// Stages include:
```

Comment:
new logic only recognizes blocks starting with `// #` as markdown

```suggestion
	// #### Stage label values
	//
	// The "stage" label can take on the following values:
```